src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export function Person({ person }) {}
+import React from 'react';
+
+export function Person({ person }) {
+  const partner = person.sex === 'm' ? 'wife' : 'husband';
```

Comment:
```suggestion
	const { sex } = person;
  const partner = sex === 'm' ? 'wife' : 'husband';
```

that way you can use your props more convenient 😉 
(just restructure the person's properties)